### PR TITLE
feat: implement rooms and talks

### PR DESCRIPTION
**Pull request by yasinuslu**

### CODE REVIEW

File: app/src/screens/scheduleScreen/talks.tsx [hunk 1]
```diff
@@ -1,15 +1,39 @@
...
+  </StyledTime>
+);
+
+const renderTalkItem: React.FC<{ item: Talk }> = ({ item }) => {
+  return (
+    <ListItem
+      icon={({ style }) => <Time style={style} time={item.startsAt} />}
+      key={item.id}
+      title={item.title}
+      description={`${formatTime(item.startsAt)} - ${formatTime(item.endsAt)}\nby ${
```
Comment (yasinuslu) [inline app/src/screens/scheduleScreen/talks.tsx]: I hate this, but it only accepts string 😄

File: app/src/types.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+export interface Room {
+  id: string;
+  name: string;
+}
+
+export interface Speaker {
+  id: string;
+  name: string;
+  email: string;
+  avatar: null;
```
Comment (guilhermewaess) [inline app/src/types.ts]: This fields with null, they can be a different value right? like string perhaps? so what if string | null?

Comment (yasinuslu) [reply app/src/types.ts]: ohh, I used the quicktype plugin to generate these, missed that one, sorry :) It's actually an image from strapi I think it's an object with some props like url. I'll update this

Comment (yasinuslu) [reply app/src/types.ts]: Fixed :D

**Pull request merged**